lib/services/provided/autocomplete.js

```diff
@@ -85,9 +84,9 @@ export function provideAutocompleteResults(
     inclusionPriority: 1,
 
     // The default provider has a suggestion priority of 1.
-    get suggestionPriority() {
-      return atom.config.get("Hydrogen.autocompleteSuggestionPriority");
-    },
+    suggestionPriority: atom.config.get(
+      "Hydrogen.autocompleteSuggestionPriority"
+    ),
```

Comment:
@wadethestealth 
Please refer to the commit message 0c5c21 for why this static `suggestionPriority` is preferable.

Reply:
I think in this one use case it is fine either way. Plus a getter this simple will not affect performance. I don't mind the change though. I just thought it was neat to not have to restart.

Reply:
Yes, the computation may be negligible, but I don't like computing `atom.config.get` every time we make completions. It maybe a talk of coding styles.

You can see other great plugins just follow the rules:
https://github.com/TypeStrong/atom-typescript/blob/master/lib/main/atom/autoCompleteProvider.ts

And the API documentation:
https://github.com/atom/autocomplete-plus/wiki/Provider-API

Reply:
And once an user set up their priority number, there wouldn't be any need for changing it dynamically.
Dynamic change would be happy only while s/he tweaking around it.

That's why I think the dynamic computation here would be really no needed.